src/astroprint/static/js/app/views/printing.js

```diff
@@ -28,7 +28,7 @@
     }
 
     if (actual !== null) {
-      this.$el.find('.temp-current').html(Math.round(actual)+'&deg;');
+      this.$el.find('sapn.temp-current').html(Math.round(actual)+' &deg;C /');
```

Comment:
is this a typo for span?

Reply:
I am not sure....I removed the code in green and changed it to the red one...

    if (actual !== null) {
      this.$el.find('.temp-current').html(Math.round(actual)+'&deg;');
    }

this is the code in the file